dartsim/src/SimulationFeatures.hh

```diff
@@ -113,6 +118,11 @@ class SimulationFeatures :
   public: std::vector<ContactInternal> GetContactsFromLastStep(
       const Identity &_worldID) const override;
 
+  public: RayIntersection GetRayIntersectionFromLastStep(
```

Comment:
Can you document here that this only works if the `bullet` detector is used?

Reply:
Done in https://github.com/gazebosim/gz-physics/pull/641/commits/add780562908eac4928fb6f260fcf3fd5f69eabc